builtin/rebase.c

```diff
@@ -852,6 +857,24 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 		break;
 	}
 
+	if (whitespace) {
+		struct strbuf buf2 = STRBUF_INIT;
+
+		strbuf_reset(&buf);
+		strbuf_reset(&buf2);
+		strbuf_addf(&buf2, "--%s", whitespace);
+		strbuf_addstr(&buf, "--whitespace");
+		sq_quote_buf(&buf, buf2.buf);
+
+		strbuf_addstr(&options.git_am_opt, buf.buf);
+
+		whitespace = xstrdup(whitespace);
+		if (strcmp(buf2.buf, "fix") || strcmp(buf2.buf, "strip"))
```

Comment:
How could buf2 be "fix" or "strip"? It is initialized above with `strbuf_addf(&buf2, "--%s", whitespace);`.

Reply:
@chriscool this is not a helpful comment. Tell @prertik what to do instead.